### PR TITLE
Add "This site is powered by Netlify" to Index footer

### PR DESCRIPTION
**Pull request by Soitora**
Last part to qualify for the [Netlify Open Source Plan](https://www.netlify.com/legal/open-source-policy), after #588.

### CODE REVIEW

**Review by arkon — APPROVED** [review]

File: src/.vuepress/theme/components/Home.vue [hunk 1]
```diff
@@ -59,8 +59,49 @@
...
-			<div v-if="data.footer" class="footer">
-				{{ data.footer }}
+			<div class="footer">
+				<div class="copyright">
+					<a href="https://www.apache.org/licenses/LICENSE-2.0">Open-source Apache Licensed</a>
+					| Copyright &copy; 2015{{ " - " + new Date().getFullYear() }} Javier Tomás
+				</div>
+				<div class="netlify">
+					<a target="_blank" href="https://www.netlify.com/">This site is powered by Netlify</a>
+					<svg
```
Comment (arkon) [inline src/.vuepress/theme/components/Home.vue:69]: Might be nice to have this in a separate file or something, dunno if that's a thing.

Comment (Soitora) [reply src/.vuepress/theme/components/Home.vue:69]: You mean like make a component for that one?

Comment (arkon) [reply src/.vuepress/theme/components/Home.vue:69]: No, more of just a separate asset. Or even just dropping it, is the logo even required?

Comment (Soitora) [reply src/.vuepress/theme/components/Home.vue:69]: Logo isn't required, but it does add to aesthetics, I can make it seperate.

**Review by arkon — APPROVED** [review]

**Pull request merged**